Bottom: 5f13632cc8cd80d3f322714cfd00ba0245c06b62
Top:    a86660274c72f2eb72ab88afaa1bcc3179db4380
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:42:41 -0500

Refresh of readme

---

diff --git a/README.md b/README.md
index aed0ed2..0b56ff6 100644
--- a/README.md
+++ b/README.md
@@ -6,6 +6,6 @@ the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc.
 
 Forked from https://github.com/pirati-cz/tablecalc. Adds the following
 functions:
-  - tag(name)     Associate the current row and column with a name
-  - tag_col(name) Return the column for the specified tag
-  - tag_row(name) Return the row for the specified tag
+  | ''tag(name)''     | Associate the current row and column with a name |
+  | ''tag_col(name)'' | Return the column for the specified tag          |
+  | ''tag_row(name)'' | Return the row for the specified tag             |
